bundles/org.openhab.binding.nibeheatpump/src/main/resources/OH-INF/thing/SMO40-types.xml

```diff
@@ -0,0 +1,9437 @@
+	</channel-type>
+	<channel-type id="smo40-48284" advanced="true">
+		<item-type>Switch</item-type>
+		<label>SG Ready hot water</label>
+		<description>Sets whether or not SG Ready should affect hot water</description>
+		<state readOnly="false"></state>
+	</channel-type>
+	<channel-type id="smo40-48285" advanced="true">
+		<item-type>Switch</item-type>
+		<label>SG Ready pool</label>
+		<description>Sets whether or not SG Ready should affect pool</description>
+		<state readOnly="false"></state>
+	</channel-type>
+	<channel-type id="smo40-48367" advanced="true">
+		<item-type>Number</item-type>
+		<label>Speed circ.pump Heat Slave 8</label>
+		<description></description>
+		<state min="1" max="100" step="1" pattern="%d %" readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="smo40-48368" advanced="true">
+		<item-type>Number</item-type>
+		<label>Speed circ.pump Heat Slave 7</label>
+		<description></description>
+		<state min="1" max="100" step="1" pattern="%d %" readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="smo40-48369" advanced="true">
+		<item-type>Number</item-type>
+		<label>Speed circ.pump Heat Slave 6</label>
+		<description></description>
+		<state min="1" max="100" step="1" pattern="%d %" readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="smo40-48370" advanced="true">
+		<item-type>Number</item-type>
+		<label>Speed circ.pump Heat Slave 5</label>
+		<description></description>
+		<state min="1" max="100" step="1" pattern="%d %" readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="smo40-48371" advanced="true">
+		<item-type>Number</item-type>
+		<label>Speed circ.pump Heat Slave 4</label>
+		<description></description>
+		<state min="1" max="100" step="1" pattern="%d %" readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="smo40-48372" advanced="true">
+		<item-type>Number</item-type>
+		<label>Speed circ.pump Heat Slave 3</label>
+		<description></description>
+		<state min="1" max="100" step="1" pattern="%d %" readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="smo40-48373" advanced="true">
+		<item-type>Number</item-type>
+		<label>Speed circ.pump Heat Slave 2</label>
+		<description></description>
+		<state min="1" max="100" step="1" pattern="%d %" readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="smo40-48374" advanced="true">
+		<item-type>Number</item-type>
+		<label>Speed circ.pump Heat Slave 1</label>
+		<description></description>
+		<state min="1" max="100" step="1" pattern="%d %" readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="smo40-48375" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Op mode circ.pump Heat Slave 8</label>
+		<description></description>
+		<state readOnly="false"></state>
+	</channel-type>
+	<channel-type id="smo40-48376" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Op mode circ.pump Heat Slave 7</label>
+		<description></descr
```

Comment:
Are the spaces in the pattern intentional?

Reply:
The label string is given by the manufacturer

Reply:
Seems like GitHub mixed up some lines in this review. I was refering to line 8110:
`<state min="-10" max="10" step="1" pattern="%d  " readOnly="false">`